### ISSUE TITLE
H14: No Web Processes Running

### ISSUE THREAD

**Issue by nky335** [virtualstaticvoid/heroku-docker-r-plumber-app#1]
When deploying to Heroku, whether using the one-push button or running the commands manually, opening the app shows an application error, and the log shows this error. Do you know what might be causing the problem?

`2020-08-30T08:19:31.263405+00:00 heroku[router]: at=error code=H14 desc="No web processes running" method=GET path="/" host=glowing-dusk-456.herokuapp.com request_id=b2a23a4d-97cc-4ad3-931b-6c426bbd230f fwd="172.255.125.136" dyno= connect= service= status=503 bytes= protocol=https`
`2020-08-30T08:19:31.880338+00:00 heroku[router]: at=error code=H14 desc="No web processes running" method=GET path="/favicon.ico" host=glowing-dusk-456.herokuapp.com request_id=7778a05a-fd4e-4159-a941-23382716d083 fwd="172.255.125.136" dyno= connect= service= status=503 bytes= protocol=https`

**Comment by virtualstaticvoid** [comment]
Hi @nky335 

I've managed to replicate and fix the issue.

The [Heroku H14 help page](https://devcenter.heroku.com/articles/error-codes#h14-no-web-dynos-running) says the error is that there aren't any running dynos and to run `heroku scale web=1` to start one.

The `heroku.yml` file in this repo names the docker build as `app` so I tried `heroku scale app=1` which started the R process successfully, however no traffic found it's way to the dyno.

I then renamed the docker build to `web`, re-deployed and manually ran `heroku scale web=1` and it worked. 

I've applied the fixes to the repo and added a `formation` configuration to the `app.json` so the manual command to scale the dynos isn't required.
